src/test/kotlin/no/nav/familie/ef/sak/beregning/OmregningServiceTest.kt

```diff
@@ -384,4 +451,65 @@ internal class OmregningServiceTest : OppslagSpringRunnerTest() {
     private fun readFile(filnavn: String): String {
         return this::class.java.getResource("/omregning/$filnavn")!!.readText()
     }
+
+    private fun lagreTilkjentYtelse(
+        behandlingId: UUID,
+        stønadsår: Int,
+    ): TilkjentYtelse {
+        val tilkjentYtelse = tilkjentYtelse(
+            behandlingId = behandlingId,
+            personIdent = "321",
+            stønadsår = stønadsår,
+            inntekt = 1,
+            beløp = 1,
+        ) // tidligere beløp betyr ikke noe
+        tilkjentYtelseRepository.insert(tilkjentYtelse)
+        return tilkjentYtelse
+    }
+
+    private fun finnInntektsperiodeEtterNyGDato(behandlingId: UUID): Inntektsperiode {
+        val behandlingNy = behandlingRepository.findByIdOrThrow(behandlingId)
+        val vedtakNy = vedtakRepository.findByIdOrThrow(behandlingNy.id)
+        return vedtakNy.inntekter?.inntekter!!.first { it.periode.inneholder(YearMonth.of(2022, 6)) }
+    }
+
+    private fun finnAndelEtterNyGDato(iverksettDto: IverksettOvergangsstønadDto) =
+        iverksettDto.vedtak.tilkjentYtelse?.andelerTilkjentYtelse?.firstOrNull {
+            it.periode.inneholder(
+                YearMonth.of(2022, 6),
+            )
+        }
+
+    private fun lagSøknadOgVilkårOgVedtak(
+        behandlingId: UUID,
+        fagsakId: UUID,
+        inntekt: Inntektsperiode,
+        stønadsår: Int,
+    ) {
+        val (fagsak, behandling) = lagreFagsakOgBehandling(fagsakId, behandlingId)
+
+        søknadService.lagreSøknadForOvergangsstønad(Testsøknad.søknadOvergangsstønad, behandling.id, fagsak.id, "1L")
+
+        val vilkårsvurderinger = lagVilkårsvurderinger(lagBarn(behandling), behandlingId)
+        vilkårsvurderingRepository.insertAll(vilkårsvurderinger)
+
+        val vedtak = vedtak(behandling.id, år = stønadsår, inntekter = InntektWrapper(listOf(inntekt)))
+        vedtakRepository.insert(vedtak)
+    }
+
+    private fun lagInntekt(dagsats: Int, månedsinntekt: Int, inntekt: Int, år: Int): Pair<Inntektsperiode, BigDecimal> {
+        val inntektPeriode = inntektsperiode(
+            år = år,
+            dagsats = dagsats.toBig(),
+            månedsinntekt = månedsinntekt.toBig(),
+            inntekt = inntekt.toBigDecimal(),
```

Comment:
Hvorfor opprette en ny util, når toBigDecimal brukes for inntekt allerede? 
```suggestion
            dagsats = dagsats.toBigDecimal(),
            månedsinntekt = månedsinntekt.toBigDecimal(),
            inntekt = inntekt.toBigDecimal(),
```

Reply:
Slettes. Tror denne var nullable på et tidlig tidspunkt - glemte rydding når dette ble endret